ideas/forms.py

```diff
@@ -18,9 +19,11 @@
 class AnonymousIdeaCreateForm(ModelForm):
     """Form for anonymous users"""
 
+    captcha = ReCaptchaField() 
+    
     class Meta:
         model = Idea
-        fields = ['title', 'concept', 'tags']
+        fields = ['title', 'concept', 'tags', 'captcha']
```

Comment:
If you could adapt the test cases for this change, it would help pass this pull-request